src/assets/scss/_settings.scss

```diff
@@ -236,6 +236,8 @@ $normal-text-font-size: 1rem;  // because the global "100% isn't useful for all
 $small-text-font-size: 0.875rem;
 $small-text-line-height: 1.357rem;
 
+$medium-text-font-size: 1.25rem;
```

Comment:
I have needed this setting SO many times! Thank you!

Reply:
Happy to put this in there, Adam requested a text size in between featured and body text. This font size is right in the middle of those two existing font types.